Vehicle.h

```diff
@@ -3,26 +3,23 @@
 #include <string>
 #include <iostream>
```

Comment:
you're not using anything in iostream here.  don't include what you don't use.
- [ ] revise